tests/sentry/hybridcloud/test_organization.py

```diff
@@ -60,7 +60,10 @@ def basic_filled_out_org() -> tuple[Organization, list[User]]:
     return org, [owner, other_user]
 
 
-def parameterize_with_orgs(f: Callable):
+WithOrgsCallback = Callable[[Callable[[], tuple[Organization, list[User]]]], None]
+
+
+def parameterize_with_orgs(f: WithOrgsCallback) -> WithOrgsCallback:
     return pytest.mark.parametrize("org_factory", [pytest.param(basic_filled_out_org)])(f)
```

Comment:
even easier than this:

```python
parameterize_with_orgs = pytest.mark.parametrize("org_factory", [pytest.param(basic_filled_out_org)])
```

if you want to stick with the function (I wouldn't recommend it) there's two easier ways to do this:

1. preserve with `ParamSpec`
```python
P = ParamSpec("P")
R = TypeVar("R")
def parametrize_with_orgs(f: Callable[P, R]) -> Callable[P, R]:
```
2. preserve type with a Callble TypeVar

```python
CallableT = TypeVar("CallableT", bound=Callable[..., object])
def parametrize_with_orgs(f: CallableT) -> CallableT:
```

Reply:
`parameterize_with_orgs = pytest.mark.parametrize("org_factory", [pytest.param(basic_filled_out_org)])` makes this so much cleaner - thank you!